File: 01-03-18/java/02-basic-of-implementation.java

import java.io.*;

class 02-basic-of-implementation
{
    public static void main(String args[] ) throws Exception {
        BufferedReader input = new BufferedReader(new InputStreamReader(System.in));
        String str = input.readLine();
        int len=str.length();
        int cnt[]=new int[10];
        for(int i=0;i<len;i++)
        {
            cnt[ str.charAt(i)-'0' ]++;
        }
        
        for(int i=0;i<10;i++)
        {
            System.out.println( i+" "+cnt[i] );
                        
        }
    }
}

File: Mar-18/01-03-18/python/02-basic-of-implementation.py
strData = input()

size = len(strData)
cnt = {}

for i in range(0,10):
    cnt[i]= int(0)
    
for i in range(0,size):
    cnt[ int(strData[i]) ]+=1
    
    
for i in range(0,10):
    print(i,cnt[i])
